src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -18,6 +18,7 @@ public sealed class JsonSerializerOptions
 
         private readonly ConcurrentDictionary<Type, JsonClassInfo> _classes = new ConcurrentDictionary<Type, JsonClassInfo>();
         private ClassMaterializer _classMaterializerStrategy;
+        private MemberAccessor _memberAcceessorStrategy;
```

Comment:
typo, extra e: `_memberAcceessorStrategy`

```suggestion
        private MemberAccessor _memberAccessorStrategy;
```